clients/imodelhub/src/IModelHubBackend.ts

```diff
@@ -7,23 +7,22 @@
  */
 
 import { join } from "path";
-import { UserCancelledError } from "./itwin-client/FileHandler";
 import { ProgressCallback, ProgressInfo } from "./itwin-client/Request";
 import {
```

Comment:
I believe that this can now be changed to `import type`, since `core-backend` is a dev dependency (not sure how it worked before).

Reply:
I don't think we can change this to `import type` since some of the imports are used for more than just their type (e.g.: `IModelHost`, `IModelJsFs`). 

If I needed to guess, `core-backend` is only dev dependency because this package (_@bentley/imodelbank-client_) is no longer published (since 3.0) and is only used internally in `full-stack-tests` and `display-test-app`.

Reply:
Correction: _@bentley/imodelbank-client_ is being published just as [internal package](https://dev.azure.com/bentleycs/iModelTechnologies/_artifacts/feed/Packages/Npm/@bentley%2Fimodelbank-client/overview/3.3.0) and from different repository: [itwinjs-internal](https://github.com/iTwin/itwinjs-internal).